pro/src/pages/Home/Homepage.tsx

```diff
@@ -223,7 +227,7 @@ export const Homepage = (): JSX.Element => {
         <BankAccountHasPendingCorrectionCallout offerer={selectedOfferer} />
       </div>
 
-      {!selectedOffererQuery.isLoading && (
+      {hasLoadOffererData && (
```

Comment:
Ça pourrait pas être un truc du genre

```suggestion
      {!selectedOffererQuery.isLoading && !(selectedOffererQuery.data || selectedOffererQuery.error) && (
```

Reply:
Alors j'avais essayé exactement ce code et aussi !selectedOffererQuery.isLoading && (selectedOffererQuery.data || selectedOffererQuery.error) mais j'avais encore le flash pour une raison obscure ...

Reply:
Je pense c'est normal, en regardant les logs aux premiers renders avec SWR isLoading est false, data est null et error est undefined. 
Tu as peut-être déjà testé aussi mais j'aurais tendance à penser qu'il faut vérifier `selectedOffererQuery.data || selectedOffererQuery.error` pour savoir si l'appel a eu lieu

Reply:
En regardant [ça](https://swr.vercel.app/docs/advanced/understanding) je me demande si `isValidating` n'est pas le bon truc à utiliser.

Reply:
En fait comme on catch l'error pour éviter le notify + retry dans ce cas là on peut pas vraiment se baser là dessus malheuresement :/